src/app/core/suggestion-notifications/qa/events/quality-assurance-event-data.service.spec.ts

```diff
@@ -0,0 +1,247 @@
+import { HttpClient } from '@angular/common/http';
```

Comment:
Starting from here, a lot of the code is under a folder named `suggestion-notifications`.  Since this is just called `Notifications` in the Admin menu, I wonder if we should consider renaming these directories to just be `notifications`? 

NOTE: If we feel this would be too difficult, I'm OK keeping this as-is.  It just seems odd to me to use the work "suggestions" as it may make this code more difficult to find for newer developers... they are more likely to search `notifications`